.github/workflows/build.yml

```diff
@@ -37,6 +37,7 @@ jobs:
 
       - name: Run Linkinator to check external links
         uses: JustinBeckwith/linkinator-action@v1
+        continue-on-error: true
```

Comment:
verwijderen uit deze branch
```suggestion
```

Reply:
@dvh dit zit ook al in main. Alleen omdat ik deze job eerst uit had gezet en daarna weer overgeschreven heb met de huidige versie in main krijg je dit te zien. Ik ga voortaan werken in atomic commits ;)